packages/eslint-config-4catalyzer-jest/index.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   overrides: [
     {
-      files: ['**/*.test.*', '**/test/**'],
+      files: ['**/*.test.*', '**/test/**', '**/__test__/**'],
```

Comment:
```suggestion
      files: ['**/*.test.*', '**/test/**', '**/__tests__/**'],
```

i believe we usually follow this pattern?

Reply:
ooops